testutil/apiload/config.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	flags "github.com/btcsuite/go-flags"
+)
+
+const (
+	DefaultResultDirectory = "results"
+	DefaultProfilesPath    = "profiles.json"
+	DefaultServer          = "http://localhost:7777"
+)
+
+type config struct {
+	ConfigPath      string `short:"c" long:"config" description:"Path to a custom configuration file."`
+	ResultDirectory string `short:"d" long:"directory" description:"Directory for the result files."`
+	ProfilesPath    string `short:"p" long:"profiles" description:"Path to custom attack profiles."`
+	Server          string `short:"s" long:"server" description:"Target base URL, with protocol."`
+	Attack          string `short:"a" long:"attack" description:"The attack to perform. The attack must match an entry in the profile definitions."`
+	ListAttacks     bool   `short:"l" long:"list" description:"List available attacks from attack profiles."`
+	FormatResults   bool   `short:"f" long:"format" description:"Pretty print the JSON result file."`
+	CPUs            int    `long:"cpus" description:"Maximum number of processors to use. Defaults to all available."`
```

Comment:
I suggest to point to the `"frequency"` parameter in profiles.json to adjust attack rate.  Also, if there was a specific use case for doing `runtime.GOMAXPROCS`, it is worth pointing it out.

Reply:
Definitely needs a frequency argument. One small caveat. The frequency in the attack definitions are per attacker. I'll create a config entry that overrides all attacker frequencies, but finer control of multi-attackers will require editing/authoring attack definitions.